### PR TITLE
feat(tooling): put excluded trigger in to not run PR for infra changes

### PR DESCRIPTION
**Pull request by m-juckes-pins**
## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: azure-pipelines-ci.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ pr:
   branches:
     include:
       - '*'
+    exclude:
+      - infrastructure
```
Comment (beejjacobs) [inline azure-pipelines-ci.yml:8]: This should be a path not a branch exclude:

```
pr:
  branches:
    include:
      - '*'
  path:
    exclude:
      - infrastructure
```
https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/pr?view=azure-pipelines#pr-autocancel-branches-paths-drafts

**Review by beejjacobs — DISMISSED** [review]

File: azure-pipelines-ci.yml (hunk 1 above)
Comment (fabfab-kc) [reply azure-pipelines-ci.yml:8]: should that be include: infrastructure?
about to put a PR for azure-pipelines-ci.yml...

`pr:
  branches:
    include:
      - '*'
  paths:
    exclude:
      - 'infrastructure'`

Comment (fabfab-kc) [reply azure-pipelines-ci.yml:8]: sorry - ignore. I thought this was for infra-ci

**Review by fabfab-kc — APPROVED** [review]
LGTM!

**Pull request merged**